Core/PrimaryGenerator.h

```diff
@@ -49,9 +53,10 @@ class PrimaryGenerator : public G4VPrimaryGenerator
   G4PrimaryVertex* GeneratePromptGammaVertex(const G4ThreeVector vtxPosition, const G4double T0, const G4double lifetimePrompt, const G4double energy);
   G4ThreeVector VertexUniformInCylinder(G4double, G4double);
   G4double calculate_mQED(Double_t mass_e, Double_t w1, Double_t w2, Double_t w3);
-  const G4ThreeVector  GetRandomPointInFilledSphere(G4double radius);
-  const G4ThreeVector  GetRandomPointOnSphere(G4double radius);
-  G4Navigator* theNavigator =  G4TransportationManager::GetTransportationManager()->GetNavigatorForTracking();
+  const G4ThreeVector GetRandomPointInFilledSphere(G4double radius);
```

Comment:
I am not sure what this const at the begining means. You meant  ` G4ThreeVector GetRandomPointInFilledSphere(G4double radius) const ;` ?

Reply:
corrected